.github/workflows/phpcs-lint.yml

```
@@ -31,11 +31,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: thenabeel/action-phpcs@v8
-        with:
-          files: "**.php,**.inc,**.module,**.install,**.profile,**.test"
-          standard: ../phpcs/Backdrop
-          fail_on_warnings: false
+      - name: Run phpcs on changed code
+        run: |
+          chmod 700 .github/misc/phpcs-changeset-check.sh
+          .github/misc/phpcs-changeset-check.sh
```

Could we pass in the parent branch here instead of always doing a check against `1.x`? We've only got one development branch right now but this might make it more flexible for PRs against PRs or branches against minor version PRs.


```suggestion
          .github/misc/phpcs-changeset-check.sh ${{ github.base_ref }}"
```

Note that `github.base_ref` variable only exists on PRs, so this script may need to skip if that variable is not set, in the event this action is run on a non-PR branch.

---

I'll think it over, but can't promise any sort of abstraction. :wink:

---

Base branch as param has been added. :wink: 
Note that the workflow file would still need adaptions, anyway (`on ... branches: [1.x]`), so I'm not sure it's particularly useful.